### ISSUE TITLE
Xportal not saving destinations between logins

### ISSUE THREAD

**Issue by Corealiss** [SpikeHimself/XPortal#53]
**Describe the bug**
after setting portal destinations, you log out an d back in and the portal saves it's previous name but no destination is set. have to set destination again and sometimes locks up game

**To Reproduce**
log out and back in and your portals will have no destination set (even after you set them  on last login.

**Expected behaviour**
should save destinations once chosen and after logging out and back in

**LogOutput file(s)**
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
XPortal.Patches.Piece_SetCreator.CheckWearNTearCreationTime (System.Boolean delayed) (at <f92d2cd1b1fe4c0888553264fc479679>:0)
XPortal.QueuedAction.Trigger () (at <f92d2cd1b1fe4c0888553264fc479679>:0)
XPortal.QueuedAction.Update () (at <f92d2cd1b1fe4c0888553264fc479679>:0)
XPortal.XPortal.Update () (at <f92d2cd1b1fe4c0888553264fc479679>:0)
**Version**
1.2.11

**Environment**
Choose from the following options where you are experiencing the problem (leave only the ones you know about):
* Multiplayer Host (my game is the server)

**Other mods**
Do you experience the issue also when you do not have any other mods installed? If applicable, please provide the full list of mods that you are using.

**Priority**
Estimate how important this problem is. Choose Medium if you don't know.

* Low (XPortal is still working fine, but sometimes a thing goes wrong)

**Additional information**
Add any addtional information here. Anything that you think might help is appreciated.

**Comment by SpikeHimself** [comment]
Hi @Corealiss, thank you very much for reporting this issue.

I have just released v1.2.12 which fixes this problem.

**Comment by Quentin-ctrL** [comment]
Hi, 

v1.2.12 makes the mod useless, it seems vanila
![Capture d’écran du 2023-06-17 15-40-03](https://github.com/SpikeHimself/XPortal/assets/70467477/272f02dd-ebc6-4238-a49f-6c61b15edaec)

[UnrestrictedPortals 1.4.2]
 [Jotunn 2.12.1]
[XPortal 1.2.12]

```

[Error  : Unity Log] FileNotFoundException: Could not load file or assembly 'MonoMod.Backports, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies.
Stack trace:
XPortal.XPortal.Awake () (at <97a7626f7d8140998e9d84c6863f514a>:0)
UnityEngine.GameObject:AddComponent(Type)
BepInEx.Bootstrap.Chainloader:Start()
UnityEngine.UI.Image:OnCanvasHierarchyChanged()

[Error  : Unity Log] IndexOutOfRangeException: Index was outside the bounds of the array.
Stack trace:
Jotunn.Managers.PieceManager.PieceTable_PrevCategory (PieceTable self) (at D:/Projects/Jotunn/JotunnLib/Managers/PieceManager.cs:966)
Jotunn.Managers.PieceManager+Patches.PieceTable_PrevCategory_Postfix (PieceTable __instance) (at D:/Projects/Jotunn/JotunnLib/Managers/PieceManager.cs:98)
(wrapper dynamic-method) PieceTable.DMD<PieceTable::PrevCategory>(PieceTable)
Player.UpdateBuildGuiInput () (at <4d120e79a991451aa6d3a1b0239df629>:0)
Player.UpdatePlacement (System.Boolean takeInput, System.Single dt) (at <4d120e79a991451aa6d3a1b0239df629>:0)
Player.Update () (at <4d120e79a991451aa6d3a1b0239df629>:0)


```

**Comment by SpikeHimself** [comment]
Hi @Quentin-ctrL, I released v1.2.13 as a hotfix, does the same issue occur there too? If not, please create a new issue.

**Comment by Quentin-ctrL** [comment]
> Hi @Quentin-ctrL, I released v1.2.13 as a hotfix, does the same issue occur there too? If not, please create a new issue.

Yes with v1.2.12 it's broken  and v1.2.13 no improvement 
see new issue https://github.com/SpikeHimself/XPortal/issues/57
